packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -115,6 +116,12 @@ class BottomSheet extends StatefulWidget {
   /// Defaults to null and falls back to [Material]'s default.
   final ShapeBorder shape;
 
+  /// {@macro flutter.widgets.Clip}
+  ///
```

Comment:
Can you explain here why someone would want to specify something aside from the Clip.none default? The PR's description should cover this as well.

Also; the shape parameter doc notes that it applies to the underlying Material widget. Might say the same thing here.

Reply:
Sure thing